Tweeter/Controllers/TweetController.cs

```diff
@@ -13,50 +13,36 @@ namespace Tweeter.Controllers
 {
     public class TweetController : ApiController
     {
-        TweeterRepository Repo = new TweeterRepository();
-
+        private TweeterRepository apiTweeterController = new TweeterRepository();
+        
         // GET api/<controller>
         public IEnumerable<Tweet> Get()
         {
-            return Repo.GetTweets();
-        }
+            var test = apiTweeterController.GetTweets();
 
-        // GET api/<controller>/5
```

Comment:
You can skip storing the `test` variable and just return the call to the repo method: `return apiTweeterController.GetTweets();`